numpy/matrixlib/defmatrix.py

```diff
@@ -1007,6 +1012,34 @@ def H(self):
     getH = H.fget
     getI = I.fget
 
+    @property
+    def S(self):
+        """
+        Regard the matrix as a scalar.
+        If `self.shape == (1, 1)` then return `self[0, 0]` as it does
+        in MATLAB. Otherwise, `raise ValueError()`
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        ret : self.dtype
+            the element of index [0, 0]
+
+        Exceptions
+        ----------
+        ValueError 
+            when `self.shape != (1, 1)`
+
+        """
+        if self.shape == (1, 1):
+            return self[0, 0]
+        else:
+            raise ValueError(f"The matrix to be regarded as scalar "
+                              "must have shape (1, 1)")
+
```

Comment:
In order to get the docstring to show in the [documentation](https://24700-908607-gh.circle-artifacts.com/0/doc/build/html/reference/generated/numpy.matrix.html?highlight=matrix#numpy.matrix), you need to add it to the [autosummary](
https://github.com/numpy/numpy/blame/main/doc/source/reference/arrays.classes.rst#L379)